client/termsetting/termsetting.ts

```diff
@@ -548,12 +548,19 @@ function setInteractivity(self) {
 		}
 		self.dom.tip.clear().showunder(self.dom.nopilldiv.node())
 		// create small menu, one option for each ele in noTermPromptOptions[]
+		const optionTip = new Menu()
 		for (const option of self.noTermPromptOptions) {
 			// {isDictionary, termtype, text, html, q{}}
 			const item = self.dom.tip.d
 				.append('div')
 				.attr('class', 'sja_menuoption sja_sharp_border')
-				.on('click', async () => {
+				.on('click', async event => {
+					optionTip.clear().hide()
+					if (option.invalid) {
```

Comment:
can this `invalid` property be documented somewhere e.g. constructor option?

Reply:
> can this `invalid` property be documented somewhere e.g. constructor option?

Does documentation in new commit look ok?